src/core/api/instance_api.cpp

```diff
@@ -65,6 +65,12 @@ otInstance *otInstanceInitMultiple(uint8_t aIdx)
 
     return instance;
 }
+
+otInstance *otInstanceGetInstance(uint8_t aIdx)
+{
+    return &Instance::Get(aIdx);
+}
```

Comment:
To fix pretty check:
```suggestion
otInstance *otInstanceGetInstance(uint8_t aIdx) { return &Instance::Get(aIdx); }
```